src/server/areImagesEqual.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = function areImagesEqual(a, b) {
+  if (a.height !== b.height) {
+    return false;
+  }
+  if (a.width !== b.width) {
+    return false;
+  }
+  const len = a.data.length;
+  for (let i = 0; i < len; i += 1) {
```

Comment:
I know you just moved this function, but we never expect any transparency in our images, so you can make this faster by incrementing by +4 every step, and only checking the first three bits (r, g, b) and skipping the fourth (a).